src/status_im/config.cljs

```diff
@@ -170,3 +170,4 @@
 (def community-accounts-selection-enabled? true)
 (def fetch-messages-enabled? (enabled? (get-config :FETCH_MESSAGES_ENABLED "1")))
 (def test-networks-enabled? (enabled? (get-config :TEST_NETWORKS_ENABLED "0")))
+(def wallet-collectible-actvity-enabled? false)
```

Comment:
we have feature flags in a name space btw 
src/status_im/feature_flags.cljs

then there's a ui in the settings to turn them on/off